cmd/kubelet/app/server.go

```diff
@@ -223,7 +223,7 @@ func NewKubeletServer() *KubeletServer {
 		RktPath:                        "",
 		RktStage1Image:                 "",
 		RootDirectory:                  defaultRootDir,
-		SerializeImagePulls:            true,
+		SerializeImagePulls:            util.BoolFlag{false, true},
```

Comment:
I think it's probably more appropriate to create an instance of a BoolFlag before the return statement, call `Default(true)`, then pass that instance here.